CIP-DappConnector/CIP-dapp-connector.md

```diff
@@ -0,0 +1,269 @@
+---
+CIP: 30
+Title: Cardano dApp-Wallet Web Bridge
+Authors: rooooooooob
+Status: Draft
+Type: Standards
```

Comment:
`Comments-URI: https://github.com/cardano-foundation/CIPs/pull/88/`
this should help people loop back to prior conversation